src/Kernel-Tests-Extended/ContextTest.extension.st

```diff
@@ -55,3 +55,17 @@ ContextTest >> testSourceNodeExecutedWhenContextIsJustAtStartpc [
 	sourceNode := context sourceNodeExecuted.
 	self assert: sourceNode equals: (self class >> testSelector) ast sendNodes first receiver
 ]
+
+{ #category : #'*Kernel-Tests-Extended' }
+ContextTest >> testWrtireVariableNamedValue [
```

Comment:
A typo:
```suggestion
ContextTest >> testWriteVariableNamedValue [
```

Reply:
I can not commit the change... ("Could not apply suggestion.") I add it to the backlog instead.